go.mod

```diff
@@ -33,11 +33,13 @@ require (
 	kubevirt.io/client-go v0.58.0
 	kubevirt.io/containerized-data-importer-api v1.53.1
 	kubevirt.io/controller-lifecycle-operator-sdk/api v0.2.4
-	kubevirt.io/qe-tools v0.1.7
+	kubevirt.io/qe-tools v0.1.8
 	kubevirt.io/ssp-operator/api v0.0.0
 	sigs.k8s.io/controller-runtime v0.13.1
 )
 
+require github.com/onsi/ginkgo/v2 v2.1.4
```

Comment:
When tables are moved to `v2`, you can remove dependency on `github.com/onsi/ginkgo` and move this line to the above `require` block.